test/semantics/assign02.f90

```diff
@@ -30,18 +30,18 @@ subroutine s1
     logical, target :: l
     real, pointer :: p
     p => r
-    !ERROR: TARGET type 'REAL(8)' is not compatible with POINTER type 'REAL(4)'
+    !ERROR: Target type REAL(8) is not compatible with pointer type REAL(4)
```

Comment:
Why no longer capitalizing the keywords?  We capitalize `REAL` and `INTEGER`.

Reply:
I have been capitalizing keywords when they refer directly to elements of Fortran source, but here "target" and "pointer" are just describing the RHS and LHS of a pointer assignment statement, not the attributes. Another example: "... the target is not an object with POINTER or TARGET attribute".

But this is kind of vague and I would be happy to have more concrete guidelines. Currently in master we have 45 messages with lower-case "pointer" in them. What should we do with "pointer assignment statement" or the RHS of one which we sometimes call the "pointer target" or "POINTER target" but not "POINTER TARGET"?

Reply:
That's fine.  There's some capitalized `POINTER` and `TARGET` keywords used in messages in call semantics checking that we should lower for consistency (not necessarily now).